apps/skilavottord/ws/src/app/modules/recyclingRequest/recyclingRequest.resolver.ts

```diff
@@ -20,7 +21,10 @@ export class RecyclingRequestResolver {
     private logger: Logger,
   ) {}
 
-  @Authorize({ throwOnUnAuthorized: false })
+  /*
+    List for all recycled vehicles
+  */
```

Comment:
Food for thought.
Comments should be treated as secondary class citizens. The code should explain itself with good names and clean code.
Unnecessary comments tend to get outdated and bring extra noise.

Reply:
So IMO these comments should be deleted since they're not needed and might get outdated.